rclcpp/include/rclcpp/timer.hpp

```diff
@@ -152,8 +154,11 @@ class TimerBase
 protected:
   Clock::SharedPtr clock_;
   std::shared_ptr<rcl_timer_t> timer_handle_;
+  rclcpp::Context::SharedPtr context_;
+  std::chrono::nanoseconds period_;
 
   std::atomic<bool> in_use_by_wait_set_{false};
+  bool autostart_{false};
```

Comment:
why adding these 3 member variables?
where are they used?
moreover `autostart_` is used only in the constructor, so no need to cache it

Reply:
Sorry, the variables were part of another implementation and forgot to delete them.